cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -28,6 +28,76 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/generate/collector"
 )
 
+const (
+	//nolint:lll
+	examples = `
+  # Using the example 'memcached-operator' and assuming a directory structure
```

Comment:
Is required to add the tree? What value does it bring?

Reply:
It isn't strictly necessary, although other commands show trees.